src/Developer.js

```diff
@@ -0,0 +1,13 @@
+export class Developer {
```

Comment:
```suggestion
export default class Developer {
```
в `index.js` вы импортируете без фигурных скобок (`import Developer from './Developer'`), поэтому добавляем `default`